back/src/forms/forms.graphql

```diff
@@ -22,7 +22,49 @@ type Query {
   forms(
     "SIRET d'un établissement dont je suis membre"
     siret: String
+
+    """
+    Nombre de bordereaux retournés.
+    Défaut à 50, maximum à 500
+    """
+    first: Int
+
+    """
+    Nombre d'éléments à ne pas récupérer en début de liste
+    Permet de paginer les résultats
+    Défaut à 0
+    """
+    skip: Int
+
+    """
+    (Optionnel) Filtre sur les statuts des bordereaux
+    Si aucun filtre n'est passé, les bordereaux seront retournés quel que soit leur statut
+    Défaut à vide.
+    """
+    status: [FormStatus]
+
     """
+    (Optionnel) Filtre sur le role de demandeur dams le bordereau
+    Par exemple:
+     - `roles: [TRANSPORTER]` renverra les bordereaux pour lesquels je suis transporteur
+     - `roles: [EMITTER, RECIPIENT]` renverra les bordereaux dont je suis l'émetteur ou le destinataire final
+    Voir `FormRole` pour la liste des roles sur lesquels il est possible de filtrer.
+    Si aucune filtre n'est passé, les bordereaux seront retournés quel que soit votre role dessus.
+    Défaut à vide.
+    """
+    roles: [FormRole]
+
+    """
+    (Optionnel) Permet de filtrer sur les bordereaux en attente d'une action de notre part
+    Si `true`, seul les bordereaux attendant une action sont renvoyés
+    Si `false`, seul les bordereaux n'attendant aucune action son renvoyés
+    Si vide, tous les bordereaux sont renvoyés
+    Défaut à vide.
+    """
+    hasNextStep: Boolean
+
+    """
+    DEPRECATED
```

Comment:
tu peux aussi utiliser la directive @deprecated(reason: ""), ça apparaitra dans la documentation

Reply:
Sur un paramètre, je ne crois pas non malheureusement

Reply:
Indeed: `Directive "deprecated" may not be used on ARGUMENT_DEFINITION`

Reply:
Ah oui en effet !